createdb.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/python
 
+from __future__ import print_function
```

Comment:
Our infra runs on RHEL7 that has a python version recent enough that we can drop these